Dockerfile

```diff
@@ -0,0 +1 @@
+FROM ubuntu:latest
```

Comment:
Using `latest` tag makes build not very reproducible, as it can start behaving differently after image is updated. For Docker it's very important IMHO

Reply:
This isn't ready for review actually, but thanks for reminding :)